comp110-worksheet-6/OxoBoard.cs

```diff
@@ -12,36 +12,75 @@ public class OxoBoard
 	{
 		// Constructor. Perform any necessary data initialisation here.
 		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+		public OxoBoard(/*int width = 3, int height = 3, int inARow = 3 */)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            playerBoard = new Mark[3, 3];
```

Comment:
You haven't declared the `playerBoard` field in your class -- this is causing compile errors which prevent your code from running